src/__tests__/fixtures/PersonsComponent.js

```diff
@@ -1,14 +1,18 @@
-import Vue from 'vue'
+import { h, defineComponent } from 'vue'
 
-export default Vue.component('persons-component', {
+export default defineComponent({
+  name: 'persons-component',
   props: {
-    persons: Array
+    persons: {
+      type: Array,
+      default: () => []
+    }
   },
-  render (h) {
-    return (
-      <ul>
-        {this.persons.map(p => <li>{p.firstName} {p.lastName}</li>)}
-      </ul>
+  render (c) {
```

Comment:
```suggestion
  render () {
```

I think this can be removed

Reply:
Whoops, thanks. Forgot to remove this after playing around with the render function.